### PR TITLE
Display our own favicon for about:addons to work around a Firefox bug

### LINKED ISSUE #10: chrome:// favicons not displayed
Probably because we don't have the authorization to load them.  
Could we hijack that and show our own set of icons?

### PR DESCRIPTION
**Pull request by ariasuni**
fix #10

### CODE REVIEW

File: src/sidebar/tab.js [hunk 1]
```diff
@@ -169,7 +169,12 @@ SideTab.prototype = {
     const imgTest = document.createElement("img");
     imgTest.src = favIconUrl;
     imgTest.onerror = () => {
-      this._resetIcon();
+      // https://bugzilla.mozilla.org/show_bug.cgi?id=1462948
+      if (favIconUrl === "chrome://mozapps/skin/extensions/extensionGeneric-16.svg") {
```
Comment (eoger) [inline src/sidebar/tab.js]: How about we do the detection at the beginning of `_setIcon` instead and set the right URL directly?

**Comment by Keith94** [comment]
Does the `extensions.svg` also benefit from the color filling in #328?

File: src/sidebar/tab.js (hunk 1 above)
Comment (ariasuni) [reply src/sidebar/tab.js]: Yes, that’s better!

**Comment by ariasuni** [comment]
@eoger I followed you suggestion :)

@Keith94 Yes!!

However, note that in order to work, the check `(favIconUrl.startsWith("chrome://") && favIconUrl.endsWith(".svg"))` that permit coloring should happen before changing URL with our own.

**Pull request merged**